models/shop_work_hours.py

```diff
@@ -35,31 +35,43 @@ def _compute_name(self):
                 end=('%.2f' % self.end_time).replace('.', ':'),
             )
 
-    name = fields.Char("Period of Time", compute="_compute_name", store=True)
-    start_time = fields.Float("Since", required=True)
-    end_time = fields.Float("To", required=True)
+    # Model Fields
+    name = fields.Char(
+        string="Period of Time",
+        compute="_compute_name",
+        store=True
```

Comment:
Here and below: Trailing comma is missing

Reply:
Done with commit [0fdb65d](https://github.com/macagua/website_shops_map/commit/0fdb65d8228ea5c2c66cb9cc341f38f5f2df6353)